godtools/App/Flows/Menu/MenuFlow.swift

```diff
@@ -318,14 +318,16 @@ class MenuFlow: Flow {
     private func getAuthErrorAlertMessage(authError: AuthErrorDomainModel) -> AlertMessageType {
         
         let localizationServices: LocalizationServices = appDiContainer.dataLayer.getLocalizationServices()
+        let appLanguageLocaleId = appDiContainer.feature.appLanguage.domainLayer.getCurrentAppLanguageUseCase().getLanguage().localeId
```

Comment:
Hey @levieggertcru, is there a way you'd prefer to get the app language here rather than instantiating the use case and pulling it out?  Not sure I know how the flows fit into clean architecture.

Reply:
The AlertMessageView has been an issue since UIAlertController requires the title and message on instantiation.  

In these situations since I really don't want to perform a sink and then call the alert message, I've been keeping a reference on the flow via Published property to keep the reference updated when the app language changes.
 
I'll either keep a reference to the app language or create a domain model that encapsulates the interface strings for the alert and keep a reference to that on flow.

In your situation I'd say it's fine to just keep a reference to the app language updated on the Flow.

Example from AppFlow (https://github.com/CruGlobal/godtools-swift/blob/develop/godtools/App/Flows/App/AppFlow.swift#L85)

Reply:
Hey @rachaelblue no problem, these are the change requests for this PR.  
- Remove ```func getLanguage() -> AppLanguageDomainModel``` in GetCurrentAppLanguageUseCase and GetAppLanguageRepositoryInterface.  We only want the single publisher method on the UseCase. 
- See my comment above on referencing appLanguage from the MenuFlow.  That will resolve the need to call getLanguage() on the UseCase since you will have a live reference to the appLanguage.

Reply:
Thanks Levi!  Made those changes.